src/components/Feed/Feed.js

```diff
@@ -75,7 +75,7 @@ export default class Feed extends Component {
     this.setState({
       loading: true
     });
-    await api.get(this.getUri(), { page })
+    await ApiClient.get(this.getUri(), { headers: page, authorized: true })
```

Comment:
Think this would have to be `{ headers: { page }, authorized: true }`. As is `options.headers` would be `1` rather than `{ page: 1 }`